matrix/app_server/llm/ray_serve_vllm.py

```diff
@@ -65,10 +65,9 @@
 try:
     from vllm.entrypoints.openai.serving_engine import (  # type: ignore[attr-defined]
         LoRAModulePath,
-        PromptAdapterPath,
     )
 except:
-    from vllm.entrypoints.openai.serving_models import LoRAModulePath, PromptAdapterPath  # type: ignore[no-redef]
```

Comment:
Is this simply deprecated or it's because it's not compatible with the gpt-oss workflow?

Reply:
we were not using PromptAdapterPath, deprecate this since this is removed in new vllm versions